db/seeds.rb

```diff
@@ -5,7 +5,8 @@
   password: 'password',
   is_home_during_day: true,
   phone: '123-555-1212',
-  transportation: 'car'
+  transportation: 'car',
+  available_from: Time.local(2022, 7, 8),
```

Comment:
To potentially future proof this, does something like `4.days.from_now` work here?

Reply:
That's a great idea! Yep, that's working - I have updated the test objects.